google/cloud/aiplatform/training_jobs.py

```diff
@@ -425,7 +523,9 @@ def _create_input_data_config(
             # create input data config
             input_data_config = gca_training_pipeline.InputDataConfig(
                 fraction_split=fraction_split,
+                filter_split=filter_split,
```

Comment:
I think there is a bug here if we end up creating multiple split configs because of the passed in arguments then only one of these splits gets set depending on however the precedence is handled by the underlying GAPIC lib.

If we use the logic described above then we should be OK because we'll only be passing one config and the rest as None.